test/parallel/test-child-process-stdio-big-write-end.js

```diff
@@ -43,10 +43,10 @@ function parent() {
   child.stdout.on('data', function(c) {
```

Comment:
If I understand correctly, the assertion on `n` will occur before any assertion by `mustCallAtLeast()`.